### ISSUE TITLE
Syntax error in filter example

### ISSUE THREAD

**Issue by Nate-Wilkins** [showdownjs/showdown#102]
https://github.com/coreyti/showdown/blob/master/README.md

The `Readme.md` file shows the following filter example but the syntax is invalid...
Which field should the anonymous function be assigned to?

```js
var demo = function(converter) {
  return [
    // Replace escaped @ symbols
    { type: 'lang', function(text) {
      return text.replace(/\\@/g, '@');
    }}
  ];
}
```

**Comment by pdeschen** [comment]
fixed by 387978acf8734469d82cf802236265486bd92ada

**Comment by Nate-Wilkins** [comment]
@pdeschen Thanks!
